udp/coder/coder_test.go

```diff
@@ -242,3 +242,16 @@ func BenchmarkUnmarshalMessage(b *testing.B) {
 		}
 	}
 }
+
+func FuzzDecode(f *testing.F) {
+	f.Add([]byte{67, 1, 0, 0, 1, 2, 3, 177, 97, 1, 98, 1, 99, 1, 100, 1, 101, 16, 255, 1})
+	f.Add([]byte{0x40, 0x1, 0x30, 0x39, 0x46, 0x77, 0x65, 0x65, 0x74, 0x61, 0x67, 0xa1, 0x3, 0xff, 'h', 'i'})
+	f.Add([]byte{88, 128, 107, 170, 134, 237, 158, 132, 150, 19, 19, 159, 72, 20, 210, 14, 23, 231, 160, 183, 145, 128, 177, 14, 82, 20, 210, 255})
+	f.Add([]byte{0x48, 0x01, 0x00, 0x00, 0xB0, 0x35, 0x4C, 0xF5, 0xD9, 0x72, 0x24, 0x0D, 0x60, 0x55, 0x6C, 0x69, 0x67, 0x68, 0x74, 0x05, 0x6C, 0x69, 0x67, 0x68, 0x74})
+	f.Add([]byte{0x48, 0x45, 0x62, 0xA2, 0x8D, 0xA2, 0x29, 0x0C, 0x18, 0x0F, 0xB5, 0x4A, 0x48, 0x5E, 0x10, 0xA3, 0x88, 0x00, 0x00, 0x00, 0x00, 0x82, 0x27, 0x10, 0x52, 0x27, 0x10, 0x61, 0x16, 0xE2, 0x06, 0xDD, 0x08, 0x00, 0x42, 0x08, 0x00, 0xFF, 0x70, 0x73, 0x9F, 0xBF, 0x62, 0x65, 0x70, 0x78, 0x1A, 0x63, 0x6F, 0x61, 0x70, 0x3A, 0x2F, 0x2F, 0x31, 0x30, 0x2E, 0x31, 0x31, 0x32, 0x2E, 0x31, 0x31, 0x32, 0x2E, 0x31, 0x30, 0x3A, 0x35, 0x37, 0x39, 0x34, 0x30, 0xFF, 0xBF, 0x62, 0x65, 0x70, 0x78, 0x1E, 0x63, 0x6F, 0x61, 0x70, 0x2B, 0x74, 0x63, 0x70, 0x3A, 0x2F, 0x2F, 0x31, 0x30, 0x2E, 0x31, 0x31, 0x32, 0x2E, 0x31, 0x31, 0x32, 0x2E, 0x31, 0x30, 0x3A, 0x34, 0x36, 0x33, 0x36, 0x33, 0xFF, 0xFF, 0xFF, 0xFF})
```

Comment:
Do these seeds influence the generated data when the TC is invoked by 

```sh
go test -fuzz=FuzzDecode -fuzztime=10s $(PWD)/udp/coder/
```
? Other TCs have 0-2 and this has 5, seems inconsistent. Asking out of curiosity, there's no problem in keeping them all.

Reply:
Yes, the seed does influence the generation of malformed data. I removed some similar-looking seed data; now the seed dataset is looking unique.

INFO: There's a little problem with go-fuzz. It can't do seed minimization based on the path explored per seed; this is always a blind hit.